From 423af209da9a3a4f2018dd97a3e38e7ed9b24e39 Mon Sep 17 00:00:00 2001
From: Ahei <aheicx@gmail.com>
Date: Fri, 19 Jul 2024 01:28:00 +0800
Subject: [PATCH] add riscv64 support

---
 macros.efi-srpm.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/macros.efi-srpm.in b/macros.efi-srpm.in
index 00503f1..635505e 100644
--- a/macros.efi-srpm.in
+++ b/macros.efi-srpm.in
@@ -45,6 +45,8 @@
   local function getaltarch()
     if arch("x86_64") then
       return("ia32")
+    elseif arch("riscv64") then
+      return("riscv64")
     else
       return("none")
     end
-- 
2.39.0.windows.2

