From 3f4b280e8578d4de3d03b4af205106813a2fdbd1 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Tue, 1 May 2018 14:44:17 -0400
Subject: [PATCH 6/6] %efi_arch: turns out %{nil} is definitely not what we
 want.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 macros.efi.in | 17 +++++------------
 1 file changed, 5 insertions(+), 12 deletions(-)

diff --git a/macros.efi.in b/macros.efi.in
index a0bd67d9b69..f4c70617ae7 100644
--- a/macros.efi.in
+++ b/macros.efi.in
@@ -48,17 +48,14 @@ elseif ifarch("%{arm}") then
 elseif ifarch("ia64") then
   print("ia64")
 else
-  print("%{nil}")
+  rpm.expand("%%{error:Not an EFI architecture}")
+  print("")
 end
 }
 
 %efi_arch_upper %{lua:
   local arch = rpm.expand("%{efi_arch}")
-  if arch == '%{nil}'
-    print(arch)
-  else
-    print(string.upper(arch))
-  end
+  print(string.upper(arch))
 }
 
 %efi_alt_arch %{lua:
@@ -75,15 +72,11 @@ end
 if ifarch("x86_64") then
   print("ia32")
 else
-  print("%{nil}")
+  print("")
 end
 }
 
 %efi_alt_arch_upper %{lua:
   local arch = rpm.expand("%{efi_alt_arch}")
-  if arch == '%{nil}'
-    print(arch)
-  else
-    print(string.upper(arch))
-  end
+  print(string.upper(arch))
 }
-- 
2.14.3

